### PR TITLE
Migration syntax

### PR DESCRIPTION
**Pull request by muazzam0x48**

### CODE REVIEW

**Comment by vijayfractl** [comment]
@muazzam0x48 Is some changes required in `agentlang/model.al`?

File: test/agentlang/test/migrations.clj [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          fp (first persons)
+          customers_select (tu/fresult (ev/eval-all-dataflows
+                                        (cn/make-instance {:Factory/LookupAll_CustomerMale {}})))
+          fcs (first customers_select)]
+      (is (and (= 5 (count customers)) (and (:Name fs) (:Age fs) (:Gender fs))))
+      (is (and (= 5 (count persons)) (and (:Name fp) (:Age fp) (:Gender fp))))
+      (is (and (= 3 (count customers_select)) (and (:Name fcs) (:Age fcs) (:Gender fcs)))))
+    (clear-model-init model-name)))
+
+(deftest test-attr-chang
```
Comment (vijayfractl) [inline test/agentlang/test/migrations.clj:51]: typo - `test-attr-change`

**Review by vijayfractl — COMMENTED** [review]
@muazzam0x48 Adding a short README in each test-model directory, explaining what is being tested will be helpful.

File: test/agentlang/test/migrations.clj (hunk 1 above)
Comment (muazzam0x48) [reply test/agentlang/test/migrations.clj:51]: I will add that, thanks for the suggestion

**Pull request merged**